src/ui/connection_profiles.ui

```diff
@@ -2237,48 +2264,6 @@ p, li { white-space: pre-wrap; }
             <property name="topMargin">
              <number>9</number>
             </property>
-            <item row="0" column="0">
```

Comment:
Same issue as before with the Qt Designer plug-in - it reorders `QGridLayout` items as they are touched and makes the changes harder to trace - give me a day or less and I can try to untangle them if that helps...